doc/dnf5_plugins/automatic.8.rst

```diff
@@ -182,7 +182,7 @@ The command email emitter configuration. Variables usable in format string argum
 ``email_to``
     list, default: ``root``
 
-    List of recipients of the message.
+    Comma-separated list of recipients of the message.
```

Comment:
My impression is that dnf5's INI-style configuration files accept the comma `,`, space ` `, and newline `\n` as delimiters for `OptionStringList`.